agent/stats/engine.go

```diff
@@ -681,5 +771,19 @@ func (engine *DockerStatsEngine) ContainerDockerStats(taskARN string, containerI
 	if !ok {
 		return nil, errors.Errorf("stats engine: container not found: %s", containerID)
 	}
-	return container.statsQueue.GetLastStat(), nil
+	containerStats := container.statsQueue.GetLastStat()
+
+	// Insert network stats in container stats
+	taskStats, ok := taskToTaskStats[taskARN]
+	if ok {
+		task, _ := engine.resolver.ResolveTaskByARN(taskARN)
+		if task.IsNetworkModeAWSVPC() {
+			taskNetworkStats := taskStats.StatsQueue.GetLastStat().Networks
+			containerStats.Networks = taskNetworkStats
+		}
+	} else {
+		seelog.Warnf("Network stats not found for container %s", containerID)
```

Comment:
i think we should skip this warning when task is not in awsvpc network mode, since it is expected that only awsvpc mode has it